mdal/api/mdal.h

```diff
@@ -120,6 +121,10 @@ MDAL_EXPORT const char *MDAL_DR_filters( DriverH driver );
 //! This may effectively load whole mesh in-memory for some providers
 //! Caller must free memory with MDAL_CloseMesh() afterwards
 MDAL_EXPORT MeshH MDAL_LoadMesh( const char *meshFile );
+
+MDAL_EXPORT void MDAL_SaveMesh( MeshH mesh, const char *meshFile );
```

Comment:
please add documentation. Specially: is this saving only mesh file or also datasets? Also maybe worth to add a function to test driver capability to do this operation (see MDAL_DR_writeDatasetsCapability)

Reply:
done.
Right now, I think it is the responsibility of the driver to handle the capability to save datasets.